### DISCUSSION TITLE
Prisma Supported Versions with the new Release

### DISCUSSION THREAD

**Discussion by cmcaboy** [graphql-nexus/nexus-prisma#392] [General]
With the new release, it isn't clear what version of Prisma is supported. Particularly, the latest version that is supported. On the release notes, there is one item that says "Support latest Prisma version". Does that mean 4.9 is supported? There is another line that says support prisma 2.30.

<img width="973" alt="image" src="https://user-images.githubusercontent.com/1489169/213752335-f05bc9f6-b76a-4264-bb98-9bf6b8be5b49.png">

Thanks!

**Comment by rostislav-simonik** [comment]
@cmcaboy when I took over of maintenance, there were already some past commits, and the change log is automatically generated during releases from past commits. So few changelog records can be confusing. 

To answer your question, the currently supported version is 4.x.x. The package build itself is still held to 4.6.1 due to an internal issue in prisma, but the already published version of nexus-prisma works well with 4.9

**Reply by cmcaboy** [reply]
Great! Thank you for the clarification!
